mod/metrics/set/type/metric/structure.rb

```diff
@@ -35,14 +37,21 @@ def right_column_class
   end
 
   view :metric_question do
-    question = subformat(card.question_card)._render_content
     wrap_with :div, question, class: "question blockquote"
   end
 
+  def question
+    subformat(card.question_card)._render_content
```

Comment:
if all we want is the plain text of the question, `card.question` will do.

render_content adds handling for nest rendering, in place editing, a div with handy css classes.  Do we want that here?

Reply:
also, standard pattern is `nest(card.question_card, view: :content)`